mscore/events.cpp

```diff
@@ -174,7 +174,11 @@ void ScoreView::wheelEvent(QWheelEvent* event)
 
       if (event->modifiers() & Qt::ControlModifier) { // Windows touch pad pinches also execute this
             QApplication::sendPostedEvents(this, 0);
+#if (QT_VERSION >= QT_VERSION_CHECK(5, 14, 1)) //???
```

Comment:
No documentation found since when this replacement exists, it does not in 5.9, and docs are for 1.14.1